tests/utils.py

```diff
@@ -0,0 +1,29 @@
+import os
+from google.cloud import bigquery
+
+# this is slow, so we evaluate this once and return the result
+try:
+    bigquery.Client()
+    BIGQUERY_AUTHENTICATED = True
+except Exception:
+    # google.auth.exceptions.DefaultCredentialsError
```

Comment:
It sounds like we know specifically what error we're expecting. If that's the case, let's catch just that. If others come up, it would be useful to have their tracebacks intsead of being eaten.